src/utils/sources-tree/getURL.js

```diff
@@ -43,19 +43,19 @@ export function getURL(sourceUrl: string, debuggeeUrl: string = ""): ParsedURL {
       // Ignore `javascript:` URLs for now
       return def;
 
-    case "webpack:":
-      // A Webpack source is a special case
+    case "moz-extension:":
+    case "resource:":
       return merge(def, {
-        path: path,
-        group: "webpack://",
-        filename: filename
+        path,
+        group: `${protocol}//${host || ""}`,
```

Comment:
flow complained `host` might be undefined, hence the failsafe.

Reply:
looking good. Can we get some more unit tests, maybe for just `getUrl`

Reply:
Sure, on it.